packages/flutter_tools/test/general.shard/resident_web_runner_test.dart

```diff
@@ -1682,13 +1692,22 @@ ResidentRunner setUpResidentRunner(FlutterDevice flutterDevice) {
   return DwdsWebRunnerFactory().createWebRunner(
     flutterDevice,
     flutterProject: FlutterProject.current(),
-    debuggingOptions: DebuggingOptions.enabled(BuildInfo.debug),
+    debuggingOptions: DebuggingOptions.enabled(_createBuildDebugInfo()),
     ipv6: true,
     stayResident: true,
     urlTunneller: null,
+    webRenderer: 'html',
   ) as ResidentWebRunner;
 }
 
+// ignore: prefer_const_constructors
```

Comment:
Why can't this be const?

Reply:
Using a const here would throw an error since we need to call `dartDefines.clear()` in debug mode.

Reply:
You should find a way to implement this without mutating the BuildInfo object, we generally assume it is unmodifiable elsewhere and I don't want to start introducing this pattern into the tooling.